modules/tensor_mechanics/src/materials/ComputeMultiPlasticityStress.C

```diff
@@ -77,6 +77,9 @@ ComputeMultiPlasticityStress::ComputeMultiPlasticityStress(const InputParameters
     _my_elasticity_tensor(RankFourTensor()),
     _my_strain_increment(RankTwoTensor())
 {
+  if (_epp_tol <= 0)
+    mooseError("ComputeMultiPlasticityStress: ep_plastic_tolerance must be positive");
+
```

Comment:
Hmm, @dschwen can probably tell us how to mess with tolerances in fparser. That's not something I'd considered when I built this system but clearly it's needed.